src/webextension/firstrun/components/welcome.js

```diff
@@ -20,6 +20,13 @@ export default function Welcome() {
         adipiscing elit. Etiam porta sem malesuada magna mollis euismod. Cras
         mattis consectetur purus sit amet fermentum.</p>
       </Localized>
+      <Localized id="welcome-warning">
+        <p><strong>Lorem ipsum dolor sit amet, consectetur.
```

Comment:
The fastest fix to to keep `<strong>` is to move `<Localized>` inside the <p>`, so that `<strong>` is an immediate child.

Reply:
Yeaaaaah, I was hoping there was a simple elegant solution. I'll drop that in for now and wait until later today to see if @jimporter has a different preference.

![screen shot 2017-11-01 at 10 59 42 am](https://user-images.githubusercontent.com/49511/32286654-cb24836e-bef3-11e7-883d-d8a7af38e355.png)


Pretty low risk here since this will likely all change and it's more important just to push this out and get it in front of any new users ASAP.